### ISSUE TITLE
Vorto repository scans only for org.eclipse.vorto.repository importers

### ISSUE THREAD

**Issue by e-grigorov** [eclipse/vorto#1100]
```
@SpringBootApplication
@EnableJpaRepositories("org.eclipse.vorto.repository")
@EntityScan("org.eclipse.vorto.repository")
@ComponentScan("org.eclipse.vorto.repository")
public class VortoRepository {
	
	public static void main(String[] args) {
		SpringApplication.run(VortoRepository.class, args);
	}
}
```
Our importer is in another domain: com.... and it's not discovered by the repository.

**Comment by aedelmann** [comment]
You can configure your importer using @Bean annotation in the Spring Configuration where you instantiate your importer manually instead of letting component-scan discover it automatically. 

But anyway, it think this problem will be resolved once your other issue which is about running importers in isolation is fixed. Then we have a consistent way how plugins (generators or importers) are registered/de-registered in the repository.

**Comment by e-grigorov** [comment]
@ComponentScan("org.eclipse.vorto.repository") - this says that only these specific packages will be scanned i.e. the configuration cannot be in com... package.

In any case, we gave a try but it failed.

It'll be helpful if you can provide a concrete working example.

**Comment by e-grigorov** [comment]
@aedelmann, that's the problematic configuration:
```
package com.foo.vorto;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class FooImporterConfiguration {
  @Bean
  public FooImporter fooImporter() {
    return new FooImporter();
  }
}
```

**Comment by aedelmann** [comment]
@SpringBootApplication
@EnableJpaRepositories("org.eclipse.vorto.repository")
@EntityScan("org.eclipse.vorto.repository")
@ComponentScan("org.eclipse.vorto.repository", "com.foo.vorto")
public class VortoRepository {
	
	public static void main(String[] args) {
		SpringApplication.run(VortoRepository.class, args);
	}
}

I know this is not ideal, since you must embed and configure the Vorto Repository with your importer. To improve this and to involve the community to build and run their own importer in an isolated way which can hook into the repository easily, see issue https://github.com/eclipse/vorto/issues/1111

**Comment by aedelmann** [comment]
Closing this issue as it will be addressed in #1111
